lib/lcevc/lcevc_dil.js

```diff
@@ -0,0 +1,281 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.lcevc.Dil');
+goog.require('shaka.log');
+goog.require('shaka.util.Dom');
+
+/**
+ * @export
+ * @summary
+ * lcevcDil provides all the operations related to the enhancement and rendering
+ * of LCEVC enabled streams and on to a canvas.
+ */
+shaka.lcevc.Dil = class {
+  /**
+   * @param {HTMLMediaElement} media The video element that will be attached to
+   * LCEVC Dil for input.
+   * @param {HTMLElement} canvas The canvas element that will be attached to
+   * LCEVC Dil to render the enhanced frames.
+   * @param {Object} dilConfig The LCEVC DIL config object
```

Comment:
This should be a more specific type.

Reply:
This is changed to `LcevcDil.LcevcDilConfig` type for clarity.